CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change log
 -----------
 
+# v2.7.2+rev3
+## (2017-10-18)
+
+* Fix the FlashEdison.json file: remove unescaped newlines in strings [Alexis]
```

Comment:
We don't need the 3 extra lines from above

Reply:
Removed